changelog/2020/JULY.md

```diff
@@ -6,6 +6,120 @@
                                 New
 --------------------------------------------------------------------------------
 
+* JUNOS
+    * Added ShowTedDatabaseIpAddress
+        * show ted database {ipaddress}
+    * Created ShowMPLSLSPNameDetail
+        * show mpls lsp name {name} detail
+    * Created ShowMPLSLSPNameExtensive
+        * show mpls lsp name {name} extensive
+    * Show Ospf3 Route Network Extensive
+        * Created ShowOspf3RouteNetworkExtensive
+    * Added ShowBFDSesssion
+        * show bfd session
+    * Added ShowBFDSesssionDetail
+        * show bfd session {ipaddress} detail
+    * Added ShowLDPSession
+        * show ldp session
+    * Added ShowClassOfService
+        * show class-of-service interface {interface}
+    * Added ShowRouteForwardingTableLabel
+        * show route forwarding-table label {label}
+    * Added ShowRSVPSession
+        * show rsvp session
+    * Added ShowRSVPNeighbor
+        * show rsvp neighbor
+    * Added ShowLdpDatabaseSessionIpaddress
+        * show ldp database session ipaddress
+    * Added ShowLdpNeighbor
+        * show ldp neighbor
+    * Added ShowRSVPNeighborDetail
+        * show rsvp neighbor detail
+    * Added ShowOspfDatabaseOpaqueArea
+        * show ospf database opaque-area
+    * Added ShowLDPInterface
+        * show ldp interface {interface}
+    * Added ShowLDPInterfaceDetail
+        * show ldp interface {interface} detail
+    * Added PingMplsRsvp
+        * ping mpls rsvp {rspv}
+    * Added TracerouteNoResolve
+        * traceroute {ipaddress} no-resolve
+    * Added Ping
+        * ping {addr} ttl {ttl} count {count} wait {wait}
 * IOSXE
-    * Added ShowStackwiseVirtual for code issue:
```

Comment:
We released 20.7 yesterday. so JULY.md was updated...

At this point, can you move your changelog to below file? currently we are adding new ones to this file toward next release. (and I knew you already talked better idea with @jeaubin on other thread)
https://github.com/CiscoTestAutomation/genieparser/blob/master/changelog/undistributed.rst